source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -119,146 +119,281 @@ corresponds to the format of the example you want to see:
+
+   .. tab:: BSON Library
+      :tabid: bson-library-read
+
+      You can read an Extended JSON string into Java objects without using
+      the MongoDB driver's document classes by using the
+      :java-docs:`JsonReader </apidocs/bson/org/bson/json/JsonReader.html>` class.
+      This class contains methods to sequentially parse the fields and values
+      in any format of the Extended JSON string, and returns them as Java objects.
+      MongoDB driver's document classes also use this class to parse Extended JSON.
+      
+      The following code example shows how you can use ``JsonReader`` to convert
+      an Extended JSON string into Java objects:
+      
+      .. code-block:: java
+      
+         String ejsonStr = "{ \"_id\": { \"$oid\": \"507f1f77bcf86cd799439011\"}," +
+                           "\"myNumber\": {\"$numberLong\": \"4794261\" }}}";
+      
+         JsonReader jsonReader = new JsonReader(ejsonStr);
+      
+         jsonReader.readStartDocument();
+      
+         jsonReader.readName("_id");
+         ObjectId id = jsonReader.readObjectId();
+         jsonReader.readName("myNumber");
+         Long myNumber = jsonReader.readInt64();
+      
+         jsonReader.readEndDocument();
+      
+         System.out.println(id + " is type: " + id.getClass().getName());
+         System.out.println(myNumber + " is type: " + myNumber.getClass().getName());
+      
+         jsonReader.close();
+      
+      The output of this code example resembles the following:
+      
+      .. code-block:: none
+         :copyable: False
+      
+         507f1f77bcf86cd799439011 is type: org.bson.types.ObjectId
+         4794261 is type: java.lang.Long
+      
+      
+      For additional information on the classes and methods mentioned on this
+      section, see the following API documentation:
+      
+   .. tab:: JsonObject Class
+      :tabid: jsonobject-class-read
+
+      You can read an Extended JSON string into a Java
+      :java-docs:`JsonObject <apidocs/bson/org/bson/json/JsonObject.html>`
+      object through the ``JsonObject`` class constructor. The ``JsonObject``
+      class acts as a wrapper for a JSON string. 
+      
+      The following example shows how to instantiate a ``JsonObject`` with an Extended JSON string:
+
+      .. code-block:: java
+      
+         String ejsonStr = "{ \"_id\": { \"$oid\": \"507f1f77bcf86cd799439011\"}," +
+                           "\"myNumber\": {\"$numberLong\": \"4794261\" }}}";
+         JsonObject jsonObj = new JsonObject(ejsonStr);
+         System.out.println(jsonObj.getJson());
+
+      The output of this code example resembles the following:
+      
+      .. code-block:: none
+         :copyable: False
+      
+         { "_id": { "$oid": "507f1f77bcf86cd799439011"},"myNumber": {"$numberLong": "4794261" }}}
+
+      If you want to work primarily with raw JSON strings and avoid intermediate ``Document`` maps in your deployment,
```

Comment:
Thanks for the feedback. I added a brief explanation of why reader might want to avoid intermediate document map, removed 'deployment' term, and added an introductory sentence at the top in migration of `JsonObject` documentation to Documents tab. https://github.com/mongodb/docs-java/pull/57/commits/67d593bb0df86f39fc0fc0fb09d4fa28420bff96#diff-f63f89b7dd8d7bbec57f0e97ee7158b8a937b8a38f1ce10186213735e45f3f8aR295-R300